src/applications/proxy-rewrite/tests/teamsite-header-footer.vrt.spec.js

```diff
@@ -0,0 +1,51 @@
+const puppeteer = require('puppeteer');
+
+const SCREEN_SIZES = [
+  ['full screen', { width: 1024, height: 800 }],
+  ['tablet screen', { width: 768, height: 1024 }],
+  ['mobile screen', { width: 375, height: 667 }],
+  ['small mobile screen', { width: 318, height: 500 }],
+];
+
+const HEADER = 'header';
+const FOOTER = '#footerNav';
+
+const TEAMSITES = [
+  ['Benefits', 'https://benefits.va.gov'],
+  ['Benefitswww', 'https://www.benefits.va.gov'],
+  ['CEM', 'https://www.cem.va.gov'],
+  ['Choose', 'https://www.choose.va.gov'],
+];
+
+describe.each(TEAMSITES)('%s', (sitename, url) => {
+  let browser;
+  let page;
+  jest.setTimeout(20000);
+
+  const compareScreenshots = async selector => {
+    const element = await page.$(selector);
+    await page.waitFor(1000);
+    const image = await element.screenshot();
+    expect(image).toMatchImageSnapshot();
+  };
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+    await page.goto(url);
+    await page.waitForSelector(FOOTER);
+    // hide foresee
+    await page.addStyleTag({ content: '.__acs { display: none !important; }' });
+  });
+
+  it.each(SCREEN_SIZES)('renders %s size', async (sreensizeName, viewport) => {
+    await page.setViewport(viewport);
```

Comment:
This could be a later enhancement, but does it make sense to click on different things and take screenshots? So we could check things like the VCL modal or the menus.

Reply:
Yeah- this is where it starts to blur the line between other tests. I think it would be nice if we could just force the app into a specific state with a redux object or simple css injection- once we start scripting interactions, this becomes combination of CSS regression + testing how the interactions work. 

That might not be a useful separation- would be good if we could prioritize these tests by failures or by working with QA to define some test cases.